model/message.go

```diff
@@ -16,11 +16,13 @@ package model
 
 // Message constants
 const (
-	CmdShell = "shell"
+	TypeShell = "shell"
 )
 
 // Message represents a message between the device and the backend
 type Message struct {
```

Comment:
I've actually been meaning to bring this up on the next BE team meeting. Since we're likely going to multiplex different protocols over the same socket in the future, wouldn't it make sense to wrap the `msgpack` packets with a fixed header?
When I was playing around with this service I redefined the message to:
```
type ProtoHdr struct {
	// ProtoType defines which protocol this message belongs to.
	ProtoType `msgpack:"typ"`
	// ContentType is an optional content type header describing
	// the protocol specific content type.
	ContentType string `msgpack:"ctyp,omitempty"`
	// Properties provide a map of optional prototype specific
	// properties (such as http headers or other meta-data).
	Properties map[string]interface{} `msgpack:"props,omitempty"`
}

// ProtoMsg is a wrapper to messages communicated on bidirectional interfaces
// such as websockets to wrap data from other application protocols.
type ProtoMsg struct {
	// Header contains a protocol specific header with a single
	// fixed ProtoType ("typ") field and optional hints for decoding 
	// the payload.
	Header ProtoHdr `msgpack:"hdr"`
	// Body contains the raw protocol data. The data contained in Body
	// can be arbitrary and must be decoded according to the protocol
	// defined in the header.
	Body []byte `msgpack:"body,omitempty"`
}
```

Reply:
I added this item it to the back-end team meeting agenda; in the meantime, I'm merging this PR as it is, as we need it to unblock the development both in UI and in mender-shell.